benchmark/benchmark_Metamorpheus.R

```diff
@@ -0,0 +1,90 @@
+library(MSstatsConvert)
+library(MSstats)
+library(parallel)
+library(stringr)
+library(jsonlite)
+library(dplyr)
+
+source("metamorpheus_Process.R")
+config <- fromJSON("scriptController.json", simplifyVector = FALSE)
+
+runBenchmarkForMetaMorpheusData <- function(datasetPath, config) {
+
+  dataset_config <- config$datasets[[datasetPath]]
+  dataset_config <- as.list(dataset_config)
+
+  cat("Processing Dataset:", dataset_config$name, "\n")
+
+  filePath <- file.path(dataset_config$parent, dataset_config$data)
+  annotPath <- dataset_config$parent
+
+  input = data.table::fread(file.path(filePath, "QuantifiedPeaks.tsv"))
+  annot = data.table::fread(file.path(annotPath, "annotation.csv"))
+
+
+  cat("Dataset File Path:", filePath, "\n")
+  cat("Annotation File Path:", annotPath, "\n")
+
+  input = input %>% filter(!str_detect(`Protein Group`, ";")) # remove multiple protein group in same cell
+  input = input %>% filter(!str_detect(`Protein Group`, "DECOY")) # remove decoys
+
+  protein_mappings = data.table::fread(file.path(filePath, "QuantifiedProteins.tsv"))
+  protein_mappings = protein_mappings %>% filter(Organism %in% c("Escherichia coli (strain K12)", "Homo sapiens"))
```

Comment:
It looks like Metamorpheus has an "Organism" column in the `QuantifiedPeaks.tsv` file now.  Could you use that?